app/lib/constants.rb

```diff
@@ -258,4 +258,21 @@ module Constants
     }
     # Dynamic values ommitted: configurable_rate_limits and social_media_handles
   }.freeze
+
+  BASE_ROLES = ["Warn",
+                "Comment Ban",
+                "Ban",
+                "Regular Member",
+                "Trusted", "Pro"].freeze
+
+  SPECIAL_ROLES = ["──────────",
```

Comment:
**nitpick (non-blocking):** Seeing this makes me think that maybe instead of adding a separator manually we could use [`grouped_options_for_select`](http://api.rubyonrails.org/classes/ActionView/Helpers/FormOptionsHelper.html#method-i-grouped_options_for_select). This would mean the following change in the view:

```ruby
<% options = { "Base Roles" => Constants::BASE_ROLES %>
<% options["Special Roles] = Constants::SPECIAL_ROLES if current_user.has_role?(:super_admin) %>
<%= f.select(:user_status, grouped_options_for_select(options), include_blank: true) %>
```